src/main/java/it/cnr/isti/hpc/wikipedia/article/Ref.java

```diff
@@ -0,0 +1,50 @@
+package it.cnr.isti.hpc.wikipedia.article;
+
+/**
+ *  This class represents inline wikipedia references. You can find the pattern to detect those inside ArticleParser.
```

Comment:
There's a way to link to other classes in Javadoc, something like `{@link ArticleParser}`, which renders the link clickable when the Javadoc is processed.